content/en/docs/concepts/containers/container-lifecycle-hooks.md

```diff
@@ -64,10 +64,20 @@ the Container ENTRYPOINT and hook fire asynchronously.
 However, if the hook takes too long to run or hangs,
 the Container cannot reach a `running` state.
 
-The behavior is similar for a `PreStop` hook.
-If the hook hangs during execution,
-the Pod phase stays in a `Terminating` state and is killed after `terminationGracePeriodSeconds` of pod ends.
-If a `PostStart` or `PreStop` hook fails,
+`PreStop` hooks are not executed asynchronously from the signal
+to stop the Container–they hook must complete its execution before
```

Comment:
```suggestion
to stop the Container; they hook must complete its execution before
```

Reply:
Thanks for the suggestion. I ended up seeing that I also made a typo on this line so I took your suggestion and fixed the typo in the latest commit.